Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -87,11 +93,43 @@ - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVide
     return nil;
 }
 
+- (BOOL)enable:(BOOL)enable videoSendingForCall:(OCTCall *)call error:(NSError **)error
+{
+    BOOL success = [self setVideoBitrate:kDefaultVideoBitRate forCall:call error:error];
+
+    if (! success) {
+        return NO;
+    }
+
+    if (enable && (call.pausedStatus == OCTCallPausedStatusNone)) {
```

Comment:
`(call.pausedStatus == OCTCallPausedStatusNone)` is used in several places. Maybe we can create category (for objcTox only, not for end user) for OCTCall with `- (BOOL)isPaused` method?

We can make this method public but I think that OCTCall API will be getting too complicated this way.